pkg/registry/core/node/strategy.go

```diff
@@ -130,7 +139,6 @@ var StatusStrategy = nodeStatusStrategy{Strategy}
 func (nodeStatusStrategy) PrepareForCreate(ctx context.Context, obj runtime.Object) {
```

Comment:
remove `nodeStatusStrategy#PrepareForCreate`... the node status API doesn't allow creation and this is confusing

Reply:
done